File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    // Get valid pyramid height from user
    do
    {
        height = get_int("Height: Type a number between 1 and 8: ");
    }
    while (height < 1 || height > 8);

    // Print the pyramid shape (height - row)
    for (int row = 1; row <= height; row++)
    {
        for (int space = 1; space <= height - row; space++)
        {
            printf(" ");
        }
        // Print left hashes of the pyramid
        for (int hash = 1; hash <= row; hash++)
        {
            printf("#");
        }
        // Print middle space
        printf("  ");

        // Print right hashes of the pyramid
        for (int hash = 1; hash <= row; hash++)
        {
            printf("#");
        }

        printf("\n");
    }
    return 0;
}

File: sentimental-mario-more/mario.py
while True:
    try:
        height = int(input("Height: Type a number between 1 and 8: "))
        if 1 <= height <= 8:
            break
    except ValueError:
        print("Please enter a valid number.")

# Print the pyramid shape (height - row)
for row in range(1, height + 1):
si
    print(" " * (height - row), end="")

    print("#" * row, end="")

    print("  ", end="")

    print("#" * row)
